### ISSUE TITLE
Rainbow Grades Tests Non-deterministic

### ISSUE THREAD

**Issue by holzbh** [Submitty/Submitty#2191]
It's been a problem for a while but I didn't find an open issue, so making one for organization. Rainbow grades tests fail because spinning up a new VM leads to very different grades despite #1090.

This will need to be completed before moving ahead with #1109, but since it's a separate problem it should be a separate issue.
